### ISSUE TITLE
Wrong VEML6075 coeffizients

### ISSUE THREAD

**Issue by al3xand3w0lf** [xinabox/xSL01#1]
where did you get these values from?
#define VEML6075_UVA_VIS_COEFF	3.33
#define VEML6075_UVA_IR_COEFF		2.5
#define VEML6075_UVB_VIS_COEFF	3.66
#define VEML6075_UVB_IR_COEFF	        2.75

Looking at Veml6075 Application Note at Page 15/22 i find different values
[designingveml6075.pdf](https://github.com/xinabox/xSL01/files/1683434/designingveml6075.pdf)

**Comment by brendanvanbreda** [comment]
hi @feuerwolf  

When looking at document you referenced, we have found out that the technical document we used as a reference is an older revision which can be found here [Vishay App Note 84339](https://www.mouser.com/pdfdocs/designingveml6075.pdf) when looking in the bottom left it can be seen as Revision: 17-Dec-15. The referenced values come for page 15/20.

As such we will update the library to reflect the latest value as per Vishay testing in updated [Vishay App Note 84339](https://www.vishay.com/docs/84339/designingveml6075.pdf) Revision 25-Nov-16.

Thanks for pointing it out.
